specification/network/resource-manager/Microsoft.Network/stable/2019-11-01/networkManagerActiveConfiguration.json

```diff
@@ -0,0 +1,240 @@
+          "description": "A unique read-only string that changes whenever the resource is updated."
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "./network.json#/definitions/SubResource"
+        }
+      ],
+      "description": "Configuration and location lists to commit."
+    },
+    "ActiveConfigurationProperties": {
+      "properties": {
+        "commits": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/CommitItem"
+          },
+          "description": "configuration Id need to deploy."
+        },
+        "deploymentTime": {
+          "type": "string",
+          "description": "Deployment time string",
+          "format": "date-time"
+        },
+        "provisioningState": {
+          "readOnly": true,
+          "$ref": "./network.json#/definitions/ProvisioningState",
+          "description": "The provisioning state of the peering Configuration resource."
+        }
+      },
+      "description": "Configuration and location lists to deploy."
+    },
+    "CommitItem": {
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/CommitsProperties",
+          "description": "The Commit properties"
+        },
+        "name": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Resource name."
+        },
+        "type": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Resource type."
+        },
+        "etag": {
+          "readOnly": true,
+          "type": "string",
+          "description": "A unique read-only string that changes whenever the resource is updated."
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "./network.json#/definitions/SubResource"
+        }
+      ],
+      "description": "Configuration and location lists to commit."
+    },
+    "CommitsProperties": {
+      "properties": {
+        "configurationType": {
+          "type": "string",
+          "description": "configuration type."
+        },
+        "securityConfiguration": {
+          "properties": {
+            "rules": {
+              "type": "array",
+              "items": {
+                "$ref": "./networkManagerSecurityConfiguration.json#/definitions/SecurityConfigurationRule"
+              },
+              "description": "configuration rule."
+            }
+          },
+          "description": "configuration Id need to deploy.",
+          "allOf": [
+            {
+              "$ref": "./networkManagerSecurityConfiguration.json#/definitions/SecurityConfiguration"
+            }
+          ]
+        },
+        "connectivityConfiguration": {
+          "description": "configuration Id need to deploy.",
+          "allOf": [
+            {
+              "$ref": "./networkManagerConnectivityConfiguration.json#/definitions/ConnectivityConfiguration"
+            }
+          ]
+        },
```

Comment:
I suppose there can be multiple connectivity configurations .. array ?

Reply:
No, for each commit, it can only contain one configuraiton